src/main/java/dave/command/FindCommand.java

```diff
@@ -0,0 +1,25 @@
+package dave.command;
+
+import dave.storage.Storage;
+import dave.task.Task;
+import dave.task.TaskList;
+import dave.ui.Ui;
+
+public class FindCommand extends Command{
+
+    public String keyword;
```

Comment:
```suggestion
    private String keyword;
```
Fields should be kept private unless the class is a data class with no behaviour.